### PR TITLE
RHOARDOC-1547: Follow-up version string update

### PR DESCRIPTION
**Pull request by inoxx03**

### CODE REVIEW

File: docs/topics/templates/document-attributes.adoc [hunk 1]
```diff
@@ -68,8 +68,8 @@ endif::[]
 // Versions
 :version-fabric8-maven-plugin: 3.5.40
```
Comment (Ladicek) [inline docs/topics/templates/document-attributes.adoc]: Should probably bump this to 4.1.0, but not sure how it affects other runtimes.

Comment (inoxx03) [reply docs/topics/templates/document-attributes.adoc]: @Ladicek I'll have to check with other runtime ENG teams to ensure they all moved to 4.1.0. I'd be inclined to make that update in a separate PR. Meanwhile, I'll redefine it to `4.1.0 ` for Thorntail.

Comment (Ladicek) [reply docs/topics/templates/document-attributes.adoc]: Works for me, thanks :-)

**Comment by tradej** [comment]
LGTM, merging. @inoxx03, please note it in the FMP 4 JIRA that the value in the Thorntail guide needs to be undefined, thanks.

**Comment by inoxx03** [comment]
@tradej Thank you, I updated the JIRA: https://issues.jboss.org/browse/RHOARDOC-1573?focusedCommentId=13725930&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13725930

**Pull request merged**